lib/login.js

```diff
@@ -109,6 +109,19 @@ module.exports = {
         debug("Page opened");
         if (status !== "success") throw new CLIError("Failed to load Azure login page!");
 
+        // Opt out of the new UI, if presented
+        const newUI = await page.evaluate(function () {
```

Comment:
I don't know how much longer they'll keep the "opt out" option open, so the long term solution should be handling the new sign in experience. This patch should work for the time being, but we should be watching out for more changes (unfortunately).

Reply:
Agree. This happened to be the shortest path to get our team authenticating again and least disruptive to your module. I'll track a backlog item for accommodating the new UI, but can't commit to actually being able to prioritize it too soon.